src/QMCDrivers/tests/test_QMCCostFunctionBatched.cpp

```diff
@@ -140,14 +141,14 @@ TEST_CASE("fillOverlapAndHamiltonianMatrices", "[drivers]")
 // the input/gold data (from a file created by convert_hdf_to_cpp.py)
 void fill_from_text(int num_opt_crowds, FillData& fd)
 {
-  // Not used in the function under test
-  int crowd_size = 1;
+  using Index = QMCTraits::IndexType;
```

Comment:
Does putting int directly work? I never find Index alias useful and  we probably should think about removing it.